lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -0,0 +1,51 @@
+# vni
+---
+all_vnis:
+  config_get: "show vni"
+  config_get_token: '/^(\d+)\s/'
+
+bridge_domain:
```

Comment:
Are bridge-domains exclusively used with vni? 
If so then fine; otherwise I think we should move this out into it's own bridge_domain.yaml file.